examples/tools/tool-package-quickstart/my_tool_package/tools/utils.py

```diff
@@ -1,29 +1,17 @@
-from pathlib import Path
-import importlib.util
 from promptflow import PFClient
 
+package_name = "my_tool_package.tools"
```

Comment:
If the "my-tools-package" reverted to the previous version using yaml, how can we make sure this package_name still work?

Reply:
I have the same concern. Looks this proposal has a special requirement for user: they need to place tool py in a separate folder from other files. Just wonder if we really need to have this limitation to users.